client/finality-grandpa/src/lib.rs

```diff
@@ -1089,32 +1095,44 @@ pub fn setup_disabled_grandpa<Block: BlockT, Client, N>(
 /// Returns the key pair of the node that is being used in the current voter set or `None`.
 fn is_voter(
 	voters: &Arc<VoterSet<AuthorityId>>,
-	keystore: &Option<KeyStorePtr>,
-) -> Option<AuthorityPair> {
+	keystore: &Option<BareCryptoStorePtr>,
```

Comment:
`&Option` feels weird here instead of `Option<&_>`. Not a new problem from this PR, but interesting...